### DISCUSSION TITLE
mongo api的代码风格不一致问题

### DISCUSSION THREAD

**Discussion by huahua132** [cloudwu/skynet#1850] [General]
```lua
function mongo_collection:createIndexes(...)

end

function mongo_collection:raw_safe_delete(delete)
	
end
```
又是驼峰又是下划线，能不能统一成下划线，看着怪难受的

**Comment by cloudwu** [comment]
不行，有历史原因。

你可以自己维护一个版本。

另外，非 bug 不要提 issue ，开 discussion

**Comment by hanxi** [comment]
可以考虑统一改一下，然后提供compat兼容旧接口。

**Comment by walking-with-linght** [comment]
> ```lua
> function mongo_collection:createIndexes(...)
> 
> end
> 
> function mongo_collection:raw_safe_delete(delete)
> 	
> end
> ```
> 
> 又是驼峰又是下划线，能不能统一成下划线，看着怪难受的

+1
